app/celery/nightly_tasks.py

```diff
@@ -160,14 +164,15 @@ def _delete_notifications_older_than_retention_by_type(notification_type):
 
     service_ids_to_purge = service_ids_that_have_sent_notifications_recently - service_ids_with_data_retention
 
-    for service_id in service_ids_to_purge:
+    for i, service_id in enumerate(service_ids_to_purge):
```

Comment:
very minor, non-blocker:

would it be useful to only stagger if
```
if len(service_ids_to_purge) > 1:
```
check otherwise it becomes a bit redundant

Reply:
Well, in the single service id case, `i` is only ever `0`, so the countdown gets set to 0 which afaik is the default.

Though you do raise an interesting point. If it's called with e.g. 3, it will still spread them over the hour. Which is perhaps a little absurd 🤷